spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbConfig.java

```diff
@@ -134,6 +141,22 @@ public boolean isUseTempFile() {
 		return this.useTempFile;
 	}
 
+	public DialectVersion getSmbMinVer() {
+		return this.smbMinVer;
+	}
+
+	public void setSmbMinVer(DialectVersion smbMinVer) {
```

Comment:
Please, add JavaDocs to new setters and mention `@since 1.2` also.

(I'm moving `master` to `1.2` meanwhile...)

Reply:
Yes will do. Also don't wait for me for the 1.2 release. This enhancement can be rolled into a future release, so no worries.

Reply:
Done!